lighthouserc.js

```diff
@@ -44,6 +45,7 @@ module.exports = {
                 'inspector-issues': 'warn',
                 'offscreen-images': 'warn',
                 'non-composited-animations': 'warn',
+                'no-document-write': 'warn',
```

Comment:
Not sure why lighthouse was complaining about this, can't find anything in my code that is related and I did a ctrl+f of the page source and I couldn't find anything related to `document.write()` 🤷. Setting it to warn for now